src/olympia/addons/views.py

```diff
@@ -284,27 +283,22 @@ def filter_paid(self):
             return self.base_queryset.top_paid(listed=False)
 
     def filter_popular(self):
-        return (self.base_queryset.order_by('-weekly_downloads')
-                .with_index(addons='downloads_type_idx'))
+        return self.base_queryset.order_by('-weekly_downloads')
```

Comment:
We should test those queries on stage, running `EXPLAIN` on them to make sure `MySQL` properly chooses the right index without our help.